duckduckgo/iOS
# hotfix: XHR abort and surrogate detection

Reviewer: Mia or Caine
Asana: 

https://app.asana.com/0/414235014887631/487711692211932
https://app.asana.com/0/414235014887631/550980311876844

CC:

**Description**:

Fixes a couple of broken sites caused by:

* XHRs not aborting (so XHRs would timeout rather than responding immediately)
* Surrogate matching did not take in to account query strings on resource URLs


**Steps to test this PR**:

1. Visit marcus.com - the site should load and render normally (rather than just be a white screen)
2. Visit fillson.com - clicking on one of the large product pictures should navigate to that product
3. Perform exploratory testing as usual
